symbolic-common/src/types.rs

```diff
@@ -769,6 +797,22 @@ impl<'a> Name<'a> {
         self.lang
     }
 
+    /// Whether the Name is mangled.
+    ///
+    /// Returns `false` for names creates with `unmangled`.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use symbolic_common::Name;
+    ///
+    /// let name = Name::unmangled("foo::bar");
+    /// assert_eq!(name.mangled(), false);
+    /// ```
+    pub fn mangled(&self) -> bool {
```

Comment:
Just because the name is not `unmangled` does not necessarily mean it is mangled. Two alternatives:

- `potentially_mangled`
- `unmangled`

On a side note, "unmangled" reads a bit strange to me. What if we call it "demangled"?